misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses_status/picking.js

```diff
@@ -73,13 +195,27 @@ const updateLineStatusFromSteps = ({ draftWFProcess, activityId, lineId }) => {
 };
 
 export const computeLineStatus = ({ draftLine }) => {
-  const stepIds = Object.keys(original(draftLine.steps));
+  const stepItems = isDraft(draftLine.steps) ? original(draftLine.steps) : draftLine.steps;
+  const stepIds = Object.keys(stepItems);
 
   if (stepIds.length > 0) {
     let countStepsCompleted = 0;
     for (let stepId of stepIds) {
+      let sumAltStepsQtysPicked = 0;
```

Comment:
imho this code shall go to some computeStepStatus function.
Here you shall assume the step status is correctly computed.